aws_codeseeder/_cfn_seedkit.py

```diff
@@ -41,10 +41,13 @@ def synth(
     subnet_ids: Optional[List[str]] = None,
     security_group_ids: Optional[List[str]] = None,
     permissions_boundary_arn: Optional[str] = None,
-) -> str:
+    synthesize: bool = False,
+    **kwargs: Dict[str, Any],
+) -> Optional[str]:
```

Comment:
I know this function doesn't have a docstring right now, but I think it would be good to add one while we're here.

I'm skeptical on using `kwargs` here -- it is two extra optional parameters, but they're completely undocumented. Why not just add `role_prefix` and `policy_prefix`?

Reply:
This is intended. `kwargs` is an escape hatch and would allow us to pass any other parameters from Seed-Farmer (or any other tool) in the future without breaking backward-compatibility.

Reply:
For what value? To decouple seed-farmer and code-seeder versions slightly?

Reply:
Yes